### ISSUE TITLE
False positive wrong-import-order for Python 3.7's dataclass

### ISSUE THREAD

**Issue by stephanlukasczyk** [PyCQA/pylint#2755]
### Steps to reproduce
1. Consider the following example (`test.py`)
```python
"""Test module for issue."""
from dataclasses import dataclass, field
from datetime import datetime
from typing import List


@dataclass
class Test:
    """A test class for issue reporting."""
    timestamp: datetime
    elements: List[str] = field(default_factory=list)


T = Test(timestamp=datetime.utcnow(), elements=["foo", "bar"])
print(T.__str__())
```
2. Run `pylint test.py`

### Current behavior
The current output of `pylint` is
```
************* Module test
test.py:3:0: C0411: standard import "from datetime import datetime" should be placed before "from dataclasses import dataclass, field" (wrong-import-order)
test.py:4:0: C0411: standard import "from typing import List" should be placed before "from dataclasses import dtaclass, field" (wrong-import-order)

-----------------------------------
Your code has been rated at 7.50/10
```

### Expected behavior
The warning should not be present (at least not when using Python 3.7+) as the `dataclasses` module is now in the standard library (cf. [PEP 557](https://www.python.org/dev/peps/pep-0557/), [What's New in Python 3.7](https://docs.python.org/3/whatsnew/3.7.html)).

### pylint --version output
```
pylint 2.2.2
asteroid 2.1.0
Python 3.7.0 (default, Jan 16 2019, 13:38:24)
[GCC 8.2.0]
```

**Comment by PCManticore** [comment]
Hi @stephanlukasczyk Thanks for the report! Can you verify what version of `isort` you have installed? For the `wrong-import-order`, `pylint` relies on `isort` to get the classification for each module, so if `isort` is not able to identify `dataclasses` as a stdlib module, then it's not necessarily a `pylint` problem. Also for some reason I cannot reproduce this locally using the latest `isort`.

**Comment by stephanlukasczyk** [comment]
Hi @PCManticore Thank you for your reply. The `isort` version installed is 4.3.4, which should be the latest, according to [its PyPI page](https://pypi.org/project/isort/).

Could it be a platform-specific issue? My system is a Gentoo Linux (with ~amd64 unmasked, latest updates installed), if you want, I could also check whether I can reproduce this issue on a MacBook tomorrow.

**Comment by PCManticore** [comment]
Sure thing, that would be useful! Do you happen to have the third party module `dataclasses` installed?

**Comment by stephanlukasczyk** [comment]
No, the third party module is not installed. I'll come back to this issue after tomorrow's tests on the MacBook, thank you for your help.

**Comment by stephanlukasczyk** [comment]
I've also checked this on the MacBook now (macOS 10.14.3) and got the same warnings.
`isort` is also version 4.3.4, the third party module `dataclasses` is not installed.
`pylint --version` output:
```
pylint 2.2.2
astroid 2.1.0
Python 3.7.2 (default, Jan 13 2019, 12:50:01)
[Clang 10.0.0 (clang-1000.11.45.5)]
```

**Comment by stephanlukasczyk** [comment]
Hi @PCManticore 
I just wanted to post a quick update, as I came back to this and checked it once again. I was now running the same code with `isort` version 4.3.10 and `pylint` version
```
pylint 2.3.1
astroid 2.2.3
Python 3.7.2 (default, Mar  4 2019, 08:12:11)
[Clang 7.0.1 (tags/RELEASE_701/final)]
```
and the warnings *disappeared*. I guess the issue was fixed somewhere on the way to this versions.
Thank you for your help!

**Comment by remort** [comment]
What if production `isort` config differs from `pylint` config.
Isort check run well, but Pylint checks fail on `wrong-import-order`, because of unconfigured Isort which Pylint uses.
How we can tell Pylint to use certain options/config when invoking `isort` ?
